pkg/systemd/terminal.jsx

```diff
@@ -51,17 +57,22 @@ const _ = cockpit.gettext;
             cockpit.user().done(function (user) {
                 this.setState({ user: user, channel: this.createChannel(user) });
             }.bind(this));
+            this.setState({ [buttonToState[this.state.theme]]: " active" });
```

Comment:
TBH I liked the previous `{ theme: name }` better. There can only ever be one active theme, and this structure allows (in principle) multiple themes to be active.